src/index.html

```diff
@@ -1,13 +1,423 @@
+                  as the most comfortable ride in town.
+                </p>
+                <span class="card__price">$ 2 590</span>
+              </article>
+            </div>
+
+            <div class="compare__item grid__item grid__item--9-12">
+              <article class="card">
+                <div class="card__image-container">
+                  <img
+                    class="card__image"
+                    src="images/photos/agileRide3.jpg"
+                    alt="sporty4"
+                  >
+                </div>
+                <h3 class="card__title">Agile ride 3</h3>
+                <p class="card__paragraph">
+                  The lightweight frame that has earned its
+                  street tread as a sleek, agile ride.
+                </p>
+                <span class="card__price">$ 2 090</span>
+              </article>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section
+        class="page__section details"
+        id="details"
+      >
+        <div class="container">
+          <h2 class="page__section-title details__title">
+            The details
+          </h2>
+
+          <article class="details__parts parts" >
+            <div class="parts__images">
+              <div class="grid">
+                <div
+                  class="grid__item grid__item--desktop-1-8 grid__item--tablet-1-4"
+                >
+                  <div class="parts__image-container">
+                    <img
+                      src="images/photos/details1.jpg"
+                      alt="part1"
+                      class="parts__image"
+                    >
+                  </div>
+                </div>
+                <div
+                  class="grid__item grid__item--desktop-9-12 grid__item--tablet-5-6"
+                >
+                  <div class="parts__image-container">
+                    <img
+                      src="images/photos/details2.jpg"
+                      alt="part1"
+                      class="parts__image"
+                    >
+                  </div>
+                </div>
+              </div>
+            </div>
+            <div class="parts__description">
+              <h3 class="parts__title">Auto Unlock</h3>
+              <p class="parts__paragraph">
+                The app senses when you're nearby to
+                unlock automatically. GPS tracking so you
+                know where your bike is and can track it anytime.
+              </p>
+            </div>
+          </article>
+
+          <article class="details__parts parts">
+            <div class="parts__images">
+              <div class="grid">
+                <div
+                  class="grid__item grid__item--desktop-1-8 grid__item--tablet-1-4"
+                >
+                  <div class="parts__image-container">
+                    <img
+                      src="images/photos/details3.jpg"
+                      alt="part1"
```

Comment:
The alt text for the image should be more descriptive, like 'Hydraulic disc brakes feature'